types/v56/types.go

```diff
@@ -2381,6 +2381,7 @@ type QueryResultVMRecordType struct {
 	DateCreated              string    `xml:"dateCreated,attr,omitempty"`
 	TotalStorageAllocatedMb  string    `xml:"totalStorageAllocatedMb,attr,omitempty"`
 	IsExpired                bool      `xml:"isExpired,attr,omitempty"`
+	Hypervisor               string    `xml:"hostName,attr,omitempty"` // HostName/hypervisor of virtual machine
```

Comment:
@thomsmoreau Though `Hypervisor` is a logical choice here, we try to keep the govcd client thin and not to rename XML fields in types. The main reason is that users look at VCD API docs and we don't want to introduce additional naming layers.

As such, please rename this field to match the XML field.

Thanks!

Reply:
Totally understandable, will do the change